### PR TITLE
Adding method to subscribe to a exchange

### PR DESCRIPTION
**Pull request by Corcioli**

### CODE REVIEW

**Comment by vitorleal** [comment]
Looks good to me  👍

File: message_queue/adapters/amqp_adapter.py [hunk 1]
```diff
@@ -171,3 +171,17 @@ def _consume_acknowledge(self, channel, tag, acknowledge=True):
...
+
+        :param function worker: Method that consume the message
+        :param string exchange: Exchange name
+        :param string exchange_type: Exchange type
+
+        """
+        self.channel.exchange_declare(exchange=exchange, exchange_type=exchange_type)
+
+        response = self.channel.queue_declare(exclusive=True)
+        self.queue = response.method.queue
```
Comment (fdiasr) [inline message_queue/adapters/amqp_adapter.py]: Check if it is possible add a prefix for queue linked with exchange

File: message_queue/adapters/amqp_adapter.py [hunk 2]
```diff
@@ -171,3 +171,17 @@ def _consume_acknowledge(self, channel, tag, acknowledge=True):
...
+        """Subscribes to a exchange.
+
+        :param function worker: Method that consume the message
+        :param string exchange: Exchange name
+        :param string exchange_type: Exchange type
+
+        """
+        self.channel.exchange_declare(exchange=exchange, exchange_type=exchange_type)
+
+        response = self.channel.queue_declare(exclusive=True)
```
Comment (fdiasr) [inline message_queue/adapters/amqp_adapter.py]: Test if it is possible scale with 2 or more workers to process it

**Comment by vanessamilagres** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;w2wx0mID&#x2F;732-worker-de-crud-de-usuario"> Worker de crud de usuário</a>

**Pull request merged**